cli/main.go

```diff
@@ -257,6 +259,74 @@ func main() {
 						return nil
 					},
 				},
+				{
+					Name:  "create",
+					Usage: "create a new image",
+					UsageText: `draupnir images create [backedUpAt] [anon.sql]
+
+[backedUpAt] an iso8601 timestamp defining when this backup was completed
+[anon.sql] path to an anonymisation script that will be run on image finalisation`,
+					Action: func(c *cli.Context) error {
+						var image models.Image
+
+						if len(c.Args()) != 2 {
+							println(c.Command.UsageText)
+							return errors.New("Invalid arguments")
+						}
+
+						backedUpAt, err := time.Parse(time.RFC3339, c.Args().Get(0))
+						if err != nil {
+							println(c.Command.UsageText)
+							return errors.Wrap(err, "Invalid backedUpAt timestamp")
+						}
+
+						anonPath := c.Args().Get(1)
+						anon, err := ioutil.ReadFile(anonPath)
+						if err != nil {
+							println(c.Command.UsageText)
+							return errors.Wrap(err, "Invalid anon script")
+						}
+
+						image, err = client.CreateImage(backedUpAt, anon)
+						if err != nil {
+							fmt.Printf("error: %s\n", err)
+							return err
+						}
+
+						fmt.Println(ImageToString(image))
+						return nil
+					},
+				},
+				{
+					Name:  "finalise",
+					Usage: "finalises an image (makes ready)",
+					UsageText: `draupnir images finalise [id]
+
+[id] the image ID to finalise- must have been previously created and uploaded`,
+					Action: func(c *cli.Context) error {
+						var image models.Image
+
+						if len(c.Args()) != 1 {
+							println(c.Command.UsageText)
+							return errors.New("Missing image ID argument")
+						}
+
+						imageID, err := strconv.Atoi(c.Args().First())
+						if err != nil {
+							println(c.Command.UsageText)
+							return errors.Wrap(err, "Invalid image ID")
+						}
+
+						image, err = client.FinaliseImage(imageID)
+						if err != nil {
+							fmt.Printf("error: %s\n", err)
```

Comment:
Could we get rid of this line and just have `return errors.Wrap(err, "Could not finalise image")`?